pkg/proxy/proxy.go

```diff
@@ -194,10 +196,12 @@ var (
 		"cilium-http-egress": {
 			proxyType: ProxyTypeHTTP,
 			ingress:   false,
+			localOnly: true,
 		},
 		"cilium-http-ingress": {
 			proxyType: ProxyTypeHTTP,
 			ingress:   true,
+			localOnly: true,
 		},
 		DNSProxyName: {
 			proxyType: ProxyTypeDNS,
```

Comment:
I would have thought DNS proxy gets the same treatment for only using local ports?